webui/react/src/components/Markdown.tsx

```diff
@@ -0,0 +1,53 @@
+import { Tabs } from 'antd';
+import { default as MarkdownViewer } from 'markdown-to-jsx';
+import React from 'react';
+
+import css from './Markdown.module.scss';
+import Spinner from './Spinner';
+
+const { TabPane } = Tabs;
+const MonacoEditor = React.lazy(() => import('components/MonacoEditor'));
+
+interface Props {
+  editing?: boolean;
+  height?: string;
+  markdown: string;
+  onChange?: (editedMarkdown: string) => void;
+}
+
+const Markdown: React.FC<Props> = ({ editing=false, height='80vh', markdown, onChange }: Props) => {
+  return (
+    editing ?<Tabs>
```

Comment:
**nit style:** for ternary JSX (not sure if there's a eslint for this) but IMO something like this a little bit easier to distinguish the true and false JSXs.
```
return editing ? (
  <Tabs>
    ...
  </Tabs>
) : (
  <div ...>
    ...
  </div>
);
```